app/main.py

```diff
@@ -16,4 +16,7 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
+    "mutable": [my_favourite_films, marks, collection_of_coins],
+    "immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info]
```

Comment:
The variable `profile_info` is incorrectly categorized as immutable. Tuples in Python are immutable, so this is correct. However, ensure that this aligns with the task requirements.